db/dcrpg/pgblockchain.go

```diff
@@ -457,17 +458,21 @@ func NewChainDBWithCancel(ctx context.Context, dbi *DBInfo, params *chaincfg.Par
 	}
 	log.Info(pgVersion)
 
-	perfSettings, err := RetrieveSysSettingsPerformance(db)
-	if err != nil {
-		return nil, err
-	}
-	log.Infof("postgres performance settings:\n%v", perfSettings)
+	// if the hidepgconfig flag was set to true then the PostgreSQL configuration
+	// settings will not be logged by default on system start up.
```

Comment:
Since the notion of a default does not exist _inside_ `NewChainDBWithCancel`, let's just say "Optionally log the PostgreSQL configuration."